bot/exts/info/resources.py

```diff
@@ -0,0 +1,47 @@
+from typing import Optional
+
+from discord import Embed
+from discord.ext import commands
+
+from bot.bot import Bot
+
+
+class Resources(commands.Cog):
+    """Display information about the Python Discord website Resource page."""
+
+    def __init__(self, bot: Bot):
+        self.bot = bot
+
+    @commands.command(name="resources", aliases=("res",))
+    async def resources_command(self, ctx: commands.Context, *, resource_topic: Optional[str]) -> None:
+        """Display information and a link to the Python Discord website Resources page."""
+        # If resource_topic isn't provided, provide link to the resources' page without any filtering of topics
+        if not resource_topic:
+            embed = Embed(
+                title="Resources",
+                description="The [Resources page](https://www.pythondiscord.com/resources) on our "
+                            "website contains a list of hand-selected learning resources that we "
+                            "regularly recommend to both beginners and experts."
+            )
+            await ctx.send(embed=embed)
+            return
+
+        # Capture everything prior to the new line allowing users to add messages below the command then prep for url
+        if len(resource_topic) > 1:
```

Comment:
Note that this more-or-less will always be True because of the if-statement you have above.

The only time it won't be True is when `resource_topic` is only 1 character.

---

I think you could remove this if-statement really.